frontend/.vscode/settings.json

```diff
@@ -82,14 +82,20 @@
     }
   ],
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
It's like you mistakenly committed the changes in this file, right?

Reply:
No, it's intentional.  Those settings changes gotta get in somehow.  :)